apps/meteor/tests/end-to-end/api/livechat/00-rooms.ts

```diff
@@ -700,6 +701,35 @@ describe('LIVECHAT - rooms', function () {
 			await deleteUser(initialAgentAssignedToChat);
 			await deleteUser(forwardChatToUser);
 		});
+
+		(IS_EE ? it : it.skip)('should return error message when transferred to a offline agent', async () => {
```

Comment:
I'm still a bit confused, is there a difference between transferring to an offline department or transferring to an offline agent? On the screen I have these two options, forward chat to a _department_ and forward chat to a _user_. 

So my question is, this is a test for transfer to an offline agent/user or forward to a department with offline agents?

Asking since when I transfer for an _offline agent_ it returns `"User is offline"` whether when I transfer to an offline department it returns `"No agents online in the department"`

So to make it short, maybe we could change the title of the test, "should return an error message when transferred to an offline department when the allowReceiveForwardOffline is disabled", or something along those lines. 

This also makes me question the name of the setting `allowReceiveForwardOffline` since it seems to only apply to departments and not agents. Let me know if this makes sense or not at all 😁